packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -156,6 +157,13 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
     selectPeople
   ]);
 
+  /* @conditional-compile-remove(rooms) */
+  const rolePermissions = _usePermissions();
+
+  let canUnmute = true;
+  /* @conditional-compile-remove(rooms) */
+  canUnmute = canUnmute && rolePermissions.microphoneButton;
```

Comment:
in conditional build it is really 
true && rolePermissions.microphoneButton 

which is really rolePermissions.microphoneButton?